{
  "Header": {
    "Original": "CVE-2025-3818 (A vulnerability, which was classified as critical, was found in webpy  ...)",
    "ID": "CVE-2025-3818",
    "Description": "(A vulnerability, which was classified as critical, was found in webpy  ...)"
  },
  "Annotations": [
    {
      "Original": "- webpy \u003cunfixed\u003e (bug #1103780)",
      "Type": "package",
      "Package": "webpy",
      "Kind": "unfixed",
      "BugNo": 1103780
    },
    {
      "Original": "NOTE: https://noppgwz8if.feishu.cn/docx/TxjpddUpTokyBwxibSgcTRr7nUf",
      "Type": "NOTE",
      "Description": "https://noppgwz8if.feishu.cn/docx/TxjpddUpTokyBwxibSgcTRr7nUf"
    },
    {
      "Original": "NOTE: https://github.com/webpy/webpy/issues/806",
      "Type": "NOTE",
      "Description": "https://github.com/webpy/webpy/issues/806"
    },
    {
      "Original": "NOTE: Fixed by: https://github.com/webpy/webpy/commit/3ba1b40e5a828a26a1df1b49cdc87395f3274c81",
      "Type": "NOTE",
      "Description": "Fixed by: https://github.com/webpy/webpy/commit/3ba1b40e5a828a26a1df1b49cdc87395f3274c81"
    }
  ]
}